packages/visual-editor/src/utils/getItem.ts

```diff
@@ -68,3 +88,17 @@ export const getItem = (
     children: tree.root.children,
   });
 };
+
+export const getItemDepthFromTree = (selector: ItemSelector, tree: CompositionTree): number => {
```

Comment:
is it possible to even generalize this further, calling it `getItemByNode` and then instead of ROOT_ID you pass in `node.id` and children would be `node.children`?